services/npm/npm.tester.js

```diff
@@ -213,7 +213,6 @@ t.create('license when network is off')
 // This tests error-handling functionality in NpmBase.
 t.create('when json is malformed for scoped package')
   .get('/l/@cycle%2Fcore.json')
-  .only()
```

Comment:
I haven't looked at the rest of this, but can we get this change in ahead of the rest of this PR. We've got other open PRS ( #1897 #1740 ) which should run against all the NPM tests.

Also, there's a couple of times a `.only()` has been missed in review. Maybe we should have a `danger` rule for this..

Reply:
Done in #1926.